explorer/nodewatch/nodewatch/RoutesFacade.py

```diff
@@ -68,15 +69,32 @@ def html_nodes(self):
 			'explorer_endpoint': self.explorer_endpoint
 		})
 
-	def json_nodes(self, role, exact_match=False):
-		"""Returns all nodes with matching role."""
+	def json_nodes(self, **kwargs):
```

Comment:
Is the ``role`` required here also?

Reply:
yes, it filters out the API or Peer node endpoint.

https://github.com/symbol/product/blob/9afd54889c3dac205e900e82dcf5c32e46b04b03/explorer/nodewatch/nodewatch/__init__.py#L117

https://github.com/symbol/product/blob/9afd54889c3dac205e900e82dcf5c32e46b04b03/explorer/nodewatch/nodewatch/__init__.py#L121

Reply:
I was asking if json_nodes should be defined as this? ``def json_nodes(self, role, **kwargs):``

Reply:
I see, it makes sense! 
I updated